SonarQube.TeamBuild.Integration/TeamBuildSettings.cs

```diff
@@ -50,6 +49,25 @@ public static class EnvironmentVariables // was internal
         #region Public static methods
 
         /// <summary>
+        /// Creates and returns settings for a non-TeamBuild environment
+        /// </summary>
+        public static TeamBuildSettings CreateNonTeamBuildSettings(string analysisBaseDirectory)
+        {
+            if (string.IsNullOrWhiteSpace(analysisBaseDirectory))
+            {
+                throw new ArgumentNullException("analysisBaseDirectory");
+            }
+
+            TeamBuildSettings settings = new TeamBuildSettings()
+            {
+                BuildEnvironment = BuildEnvironment.NotTeamBuild,
+                AnalysisBaseDirectory = analysisBaseDirectory,
+            };
+
+            return settings;
+        }
+
+        /// <summary>
```

Comment:
Added a separate constructor, primarily for test purposes, that creates a valid non-TeamBuild settings instance.